### PR TITLE
[MINOR][PySpark][DOCS] Fix examples in PySpark documentation

### PR DESCRIPTION
**Pull request by HyukjinKwon**
## What changes were proposed in this pull request?

This PR proposes to fix wrongly indented examples in PySpark documentation

```
-        >>> json_sdf = spark.readStream.format("json")\
-                                       .schema(sdf_schema)\
-                                       .load(tempfile.mkdtemp())
+        >>> json_sdf = spark.readStream.format("json") \\
+        ...     .schema(sdf_schema) \\
+        ...     .load(tempfile.mkdtemp())
```

```
-        people.filter(people.age > 30).join(department, people.deptId == department.id)\
+        people.filter(people.age > 30).join(department, people.deptId == department.id) \\
```

```
-        >>> examples = [LabeledPoint(1.1, Vectors.sparse(3, [(0, 1.23), (2, 4.56)])), \
-                        LabeledPoint(0.0, Vectors.dense([1.01, 2.02, 3.03]))]
+        >>> examples = [LabeledPoint(1.1, Vectors.sparse(3, [(0, 1.23), (2, 4.56)])),
+        ...             LabeledPoint(0.0, Vectors.dense([1.01, 2.02, 3.03]))]
```

```
-        >>> examples = [LabeledPoint(1.1, Vectors.sparse(3, [(0, -1.23), (2, 4.56e-7)])), \
-                        LabeledPoint(0.0, Vectors.dense([1.01, 2.02, 3.03]))]
+        >>> examples = [LabeledPoint(1.1, Vectors.sparse(3, [(0, -1.23), (2, 4.56e-7)])),
+        ...             LabeledPoint(0.0, Vectors.dense([1.01, 2.02, 3.03]))]
```

```
-        ...      for x in iterator:
-        ...           print(x)
+        ...     for x in iterator:
+        ...          print(x)
```
## How was this patch tested?

Manually tested.

**Before**

**After**

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
Could you take a look @srowen ? I made this change before across PySpark documentation but it seems I missed one before and then another one is introduced somewhere after my previous PR.

File: python/pyspark/sql/streaming.py [hunk 1]
```diff
@@ -315,9 +315,9 @@ def load(self, path=None, format=None, schema=None, **options):
         :param schema: optional :class:`pyspark.sql.types.StructType` for the input schema.
         :param options: all other string options
 
-        >>> json_sdf = spark.readStream.format("json")\
-                                       .schema(sdf_schema)\
-                                       .load(tempfile.mkdtemp())
+        >>> json_sdf = spark.readStream.format("json") \\
```
Comment (srowen) [inline python/pyspark/sql/streaming.py]: I see, does the \\ vs \ make the difference here? are there possibly other instances of this?

Comment (HyukjinKwon) [inline python/pyspark/sql/streaming.py]: Yes, it seems it does. I took a look and a sweep before - https://github.com/apache/spark/pull/14063

I will take another look across the documentation again tomorrow.

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by HyukjinKwon** [comment]
@srowen I just took a scan twice and I think they should be all.

**Comment by HyukjinKwon** [comment]
Let me update the PR description just in case.

**Comment by HyukjinKwon** [comment]
(PR description is updated again)

**Comment by srowen** [comment]
Merged to master/2.0

**Pull request closed without merging**